app/requirements/development.txt

```diff
@@ -2,9 +2,84 @@
 
 mypy==0.961
 yesqa==1.3.0
+alembic==1.8.1
```

Comment:
You don't need to add the dependencies here (in the dev requirements) since you've already added them to the base requirements. You should leave this file unchanged